test/exercises_test.dart

```diff
@@ -27,14 +27,37 @@ Future runCmd(List<String> cmds) async {
   assert(res.exitCode == 0);
 }
 
+Future getPackageName() async {
+  final pubspec = new File("pubspec.yaml");
+
+  final packageName = loadYaml(await pubspec.readAsString())["name"];
+
+  return packageName;
+}
+
+Future locateExercismDirAndExecuteTests() async {
+  final exercisesRootDir = new Directory("exercises");
+
+  assert(await exercisesRootDir.exists());
+
+  final exercisesDirs = exercisesRootDir.listSync().where((d) => d is Directory);
+
+  /// Sort directories alphabetically
+  final sortedExerciseDirs = exercisesDirs.toList();
+  sortedExerciseDirs.sort((a, b) => a.path.compareTo(b.path));
```

Comment:
Why is there a need to sort the list?

Reply:
No need, just thought it would be easier to look for tests in the output on travis-ci  or when running locally.